iphone/Maps/Core/iCloud/SynchronizationFileWriter.swift

```diff
@@ -80,10 +80,10 @@ final class SynchronizationFileWriter {
   }
 
   private func removeFromLocalContainer(_ cloudMetadataItem: CloudMetadataItem, completion: @escaping WritingResultCompletionHandler) {
-    LOG(.debug, "Start removing file \(cloudMetadataItem.fileName) from the local directory...")
+    LOG(.info, "Start removing file \(cloudMetadataItem.fileName) from the local directory...")
     let targetLocalFileUrl = cloudMetadataItem.relatedLocalItemUrl(to: localDirectoryUrl)
     guard fileManager.fileExists(atPath: targetLocalFileUrl.path) else {
-      LOG(.debug, "File \(cloudMetadataItem.fileName) doesn't exist in the local directory and cannot be removed.")
+      LOG(.warning, "File \(cloudMetadataItem.fileName) doesn't exist in the local directory and cannot be removed.")
```

Comment:
Is it the case when file was deleted locally?

Reply:
This is an additional check that the local file exists before deleting the file by the bookmarks manager.
```swift
  private func removeFromLocalContainer(_ cloudMetadataItem: CloudMetadataItem, completion: @escaping WritingResultCompletionHandler) {
    LOG(.debug, "Start removing file \(cloudMetadataItem.fileName) from the local directory...")
    let targetLocalFileUrl = cloudMetadataItem.relatedLocalItemUrl(to: localDirectoryUrl)
    guard fileManager.fileExists(atPath: targetLocalFileUrl.path) else {
      LOG(.debug, "File \(cloudMetadataItem.fileName) doesn't exist in the local directory and cannot be removed.")
      completion(.success)
      return
    }
    completion(.deleteCategoriesAtURLs([targetLocalFileUrl]))
    LOG(.debug, "File \(cloudMetadataItem.fileName) is removed from the local directory successfully.")
  }
```